posts/taking-learning-outside-the-classroom-into-the-cloud.md

```diff
@@ -0,0 +1,54 @@
+---
+authors:
+- Emily Slaughter
+tags:
+- Culture
+- Internship
+- Student
+- LifeAtIppon
+date: 20121-04-29T17:00:00.000Z
+title: "Taking Learning Outside of the Classroom and Into the Cloud"
+image: 
+---
+
+A love for learning is a quality that I have always looked for in peers and teammates, and it’s one that I’ve had the opportunity and privilege to invest in daily over the past nine months. To me, a commitment to learning requires humility to accept that we often have far more questions than answers to a problem. But when we commit ourselves to learning, we are promising to always seek self-improvement: to honor our questions, seek better understanding, and ultimately be a better version of ourselves today than we were yesterday. In the workplace, a love for learning means we are honest and accountable with our teammates, and more importantly, we support and believe in each other’s growth. It was this exact quality that drew me to working with Ippon Technologies USA.
+
+In fact, it was around this time last year that I accepted an offer to intern over the summer with Ippon in the Richmond, VA office. Thrilled to return to my hometown for the summer, I also marveled at the idea of finally getting hands-on experience in the software industry with talented folks whose values aligned with mine. However, as was the case with the rest of the world, the global pandemic quickly uprooted my plans for the future. Faced with growing uncertainty about the year ahead, I decided to delay my senior year of college, and I was extremely fortunate to extend my remote internship with Ippon into a full-time student software engineer position.
+
+I cannot express enough gratitude to the many mentors who have taught me over the past nine months, especially my managers Ben Scott and Amine Ouali Alami, whose humility and leadership I can only hope to emulate one day. This blog post chronicles my nine-month deep-dive into the industry and reflects on the challenges I’ve encountered, as well as the lessons I’ve learned.
```

Comment:
```suggestion
I cannot express enough gratitude to the many mentors who have taught me over the past nine months, especially my managers [Ben Scott](https://www.linkedin.com/in/benscott-agile-coach/) and [Amine Ouali Alami](https://www.linkedin.com/in/benscott-agile-coach/), whose humility and leadership I can only hope to emulate one day. This blog post chronicles my nine-month deep-dive into the industry and reflects on the challenges I’ve encountered, as well as the lessons I’ve learned.
```

Reply:
I would say [Amine Ouali Alami](https://www.linkedin.com/in/amine-ouali/)

Reply:
> 
Correct Linkedin   https://www.linkedin.com/in/amine-ouali/